addon/classes/Row.js

```diff
@@ -33,6 +35,17 @@ export default class Row extends Ember.ObjectProxy.extend({
    * @type {String | Array}
    */
   classNames: null,
+
+  /**
+   * Element ID for the corresponding `{{lt-row}}`.
+   *
+   * @property elementId
+   * @type {String}
+   * @readOnly
+   */
+  elementId: computed(function() {
```

Comment:
Lets keep this as `id` and just return the guid. This should be unique enough. I just dont think any property here should directly correlate to a component.